app/src/main/java/ch/epfl/sdp/peakar/user/profile/NewProfileActivity.java

```diff
@@ -80,27 +122,30 @@ private void hideUI(boolean self, boolean friends) {
 
     /**
      * Fill list view.
-     * TODO get list from DB?
+     * TODO show correct points and correct date
      */
     private void fillListView() {
-        ArrayList<NewCollectedItem> items = new ArrayList<>(Arrays.asList(
-                new NewCollectedItem("Mont Blanc - Monte Bianco", 4810000, 4810,
-                        true, 0, 0, "2000-01-01"),
-                new NewCollectedItem("Diablerets", 1210000, 3210,
-                        false, 0, 0, "2000-01-01"),
-                new NewCollectedItem("Kebnekaise", 710000, 2097,
-                        true, 0, 0, "2000-01-01")
-        ));
-        //ArrayList<NewCollectedItem> items = new ArrayList<>();
+        ArrayList<NewCollectedItem> newItems = new ArrayList<>();
+        for(POIPoint discoveredPeak: AuthService.getInstance().getAuthAccount().getDiscoveredPeaks()) {
+            NewCollectedItem newCollectedItem = new NewCollectedItem(
+                    discoveredPeak.getName(),
+                    0,
```

Comment:
is the points not saved currently in `discoveredPeak`?

Reply:
Exactly.